shared_model/bindings/bindings.i

```diff
@@ -51,6 +54,19 @@ namespace std {
 %rename(kp_equal) shared_model::crypto::Keypair::operator==;
 %rename(hash_invoke) shared_model::crypto::Hash::Hasher::operator();
 
+%ignore shared_model::interface::PermissionSet::PermissionSet(std::initializer_list<shared_model::interface::permissions::Role>);
+%ignore shared_model::interface::PermissionSet::PermissionSet(std::initializer_list<shared_model::interface::permissions::Grantable>);
+
+%extend shared_model::interface::PermissionSet {
+  PermissionSet(const std::vector<int> &perms) {
+    shared_model::interface::PermissionSet<Perm> *set = new shared_model::interface::PermissionSet<Perm>;
```

Comment:
I'm a bit worried about this constructor. Does garbage collector in client languages able to correctly remove such objects?

Reply:
Sure, swig only support managed languages (afair)